packages/paste-design-tokens/tokens/global/line-height.yml

```diff
@@ -0,0 +1,34 @@
+global:
+  category: line-height
+  type: number
+props:
+  line-height-10:
+    value: "1.67" ##12px * 1.67 = 20px
```

Comment:
The ordering of these is weird.  Is that because they map 1:1 to fontSizes?

Reply:
Would this be a breaking change?

Reply:
Yeah, because we use unitless values, we have to have a 1:1 mapping to the corresponding font size otherwise we won't get whole numbers for line-height. The line-height scale is based on font-size*1.5 rounded _up_ to the nearest 4

Reply:
It's not a breaking change in that we never had any, and the values being replaced in the hard coded theme object were based on the old typographic scale, but the new ones aren't too far off

Reply:
We had some in paste-theme-tokens/src/default/index.ts

<img width="516" alt="image" src="https://user-images.githubusercontent.com/1592327/63127743-fab1be00-bf67-11e9-90ce-8c3c87923dfb.png">

Which are being used already like in the website breadcrumb component

Reply:
Fixed up @TheSisb. Each font size now uses the correct line-height token in this branch.